cmake/config/gbs-tizen.cmake

```diff
@@ -0,0 +1,18 @@
+# Copyright 2018-present Samsung Electronics Co., Ltd. and other contributors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+include(CMakeForceCompiler)
+
+set(CMAKE_SYSTEM_NAME Tizen)
+set(CMAKE_SYSTEM_PROCESSOR armv7l)
```

Comment:
@haesik What about `arm-tizen-gbs.conf` to conform naming convention? I prefer to use `arm-tizen.cmake` and handle both of gbs and native build in same file. I would remove native build code if we don't support it any more.

Reply:
@haesik I prefer to use `arm-tizen.cmake` that contains gbs build only. Because gbs is the official way to build tizen package. But if there is reason to support both of gbs and non-gbs (ordinary arm linux binary), `arm-tizen-gbs.conf` would be better to specify that it is for gbs build.

Next, when we prepared to support x86 in future, I still prefer to put x86-tizen and arm-tizen in `x86-tizen.conf` and `arm-tizen.conf` respectively. Because it is more clear and other existing config follow the convention (e.g., `x86_64-linux.conf` and `arm-linux.conf`) in same directory. You may merge two files into one file under whatever you name for it. But for now, it is only for arm/tizen (with gbs).

Reply:
I don't agree with your opinion of specifying arm or x86.. if we use ar-tizen.conf x86-tizen.conf, we should make arm-tizen.conf, x86-tizen.conf, x86_64-tizen, etc.. for the same gbs build. But I agree to use noarch.. so, ok I will use noarch instead of gbs.